cmd/influx_tools/compact/command.go

```diff
@@ -62,39 +61,26 @@ func (cmd *Command) Run(args []string) (err error) {
 		return err
 	}
 
-	err = cmd.server.Open(cmd.configPath)
-	if err != nil {
-		return err
-	}
-	defer cmd.server.Close()
-
-	err = cmd.openStore()
-	if err != nil {
-		return err
-	}
-	defer cmd.closeStore()
-
-	if sh := cmd.getShard(); sh == nil {
-		return fmt.Errorf("shard %d does not exist", cmd.shardID)
-	} else if sh.IsIdle() {
-		fmt.Printf("shard %d is fully compacted\n", cmd.shardID)
-		return nil
+	var log = zap.NewNop()
+	if cmd.verbose {
+		cfg := logger.Config{Format: "logfmt"}
+		log, err = cfg.New(os.Stdout)
+		if err != nil {
+			return err
+		}
 	}
 
-	dataPath := filepath.Join(cmd.server.TSDBConfig().Dir, cmd.database, cmd.rp)
-	shardDataPath := filepath.Join(dataPath, strconv.Itoa(int(cmd.shardID)))
-	walPath := filepath.Join(cmd.server.TSDBConfig().WALDir, cmd.database, cmd.rp)
-	shardWalPath := filepath.Join(walPath, strconv.Itoa(int(cmd.shardID)))
+	fmt.Printf("opening shard at path %q\n\n", cmd.path)
```

Comment:
I think maybe we should use `fmt.FprintX` with `cmd.Stdout` so it's easier to add options to change the output.